src/actionExecutor.ts

```diff
@@ -95,15 +96,15 @@ export class ActionExecutor {
         executedAction => executedAction.href === action.href
       )
     ) {
-      log(`Cannot execute action twice: ${JSON.stringify(action)}!`);
+      logger.debug(`Cannot execute action twice: ${JSON.stringify(action)}!`);
```

Comment:
🙃 For me, debug is used to print variable names, states, etc. and trace used for displaying what happened (tracing the program flow). Feel free to ignore though. 

```suggestion
      logger.trace(`Cannot execute action twice: ${JSON.stringify(action)}!`);
```

Reply:
when we discussed the differences between `trace`, `info` and `debug` we defined the rule that `trace` should be process flow, `info` adds information about a specific object and `debug` adds more details that are important to debug. 
But in a way, in the end it is also important to give the user information when he needs it. 
I would leave this on debug, because it might be good for a user to see at this point. This is not purely process flow related here.